src/webrtc/groupCall.ts

```diff
@@ -1461,4 +1474,19 @@ export class GroupCall extends TypedEventEmitter<
             );
         }
     };
+
+    /**
+     * allowCallWithOutVideoAndAudio should be a GroupCall property. However, I make it as runtime configurable
+     * property because this additional behavior has a big impact on other features and is a big API change.
+     * Joining without a camera and video should only be explicitly allowed, because there can be unexpected behavior
+     * when later querying the media access rights from browser that the client cannot intercept. Join without came
+     * and audio should be a property like PTT.
+     */
+    public get initCallWithoutVideoAndAudio(): boolean {
+        return this.allowCallWithOutVideoAndAudio;
+    }
+
+    public set initCallWithoutVideoAndAudio(allowCallWithOutVideoAndAudio: boolean) {
+        this.allowCallWithOutVideoAndAudio = allowCallWithOutVideoAndAudio;
```

Comment:
Why not put this into opts and make it readonly and public?

Reply:
In the beginning `createGroupCall` from the MatrixClient stopped me from doing that. But since the configuration is now done via the Matrix Client, there is no reason not to do it via the constructor. Thx :+1: